src/Sulu/Bundle/AdminBundle/Resources/js/components/Checkbox/Checkbox.js

```diff
@@ -5,18 +5,18 @@ import Switch from '../Switch';
 import type {SwitchProps} from '../Switch';
 import checkboxStyles from './checkbox.scss';
 
-type Props = SwitchProps & {
+type Props = {|
```

Comment:
i refactored the component to use an `exact object type` here. is there a reason for not using `exact object types` for component props?

Reply:
Didn't know that the spread operator works in flow types, if it works this way, and all the type checks are correctly working this is even better than the current solution IMO.

The problem was that the `&` operator used before didn't do well with exact objects (which somehow also makes sense).

The other thing is that we didn't know about exact objects at the very beginning :slightly_smiling_face:  So if you encounter a component where no exact objects are used, and it would be easily possible, feel free to adapt it.